framework/test-vectors/decryption-manifest.md

```diff
@@ -0,0 +1,94 @@
+
+## Dependencies
+
+This serves as a reference of all features that this feature depends on.
+
+| Feature                                  | Min Version | Max Version |
+| ---------------------------------------- | ----------- | ----------- |
+| [keys-description](./key-description.md) | 1           | n/a         |
+| [keys-manifest](./keys-manifest.md)      | 1           | n/a         |
+
+## Summary
+
+The decrypt manifest describes static test vectors.
+This includes all the necessary details such as keyrings, cmm, encryption context, algorithm suite, etc.
+
+## Out of Scope
+
+This file is not a description of how the test vectors that will be generated by a compatible client should be decrypted.
+
+## Motivation
+
+We need a way of describing all scenarios that we want to cover with test vectors.
+This will be used for generating test vectors with an unknown implementation
+to be tested for compatibility using a known good implementation.
+It will also help us reason about the correctness of specific features or keyrings.
+It can also used to ensure known bad vectors or incorrect configuration cannot lead to successful decryption
+by demonstrating that different configurations succeed or fail correctly.
+
+## Guide-level Explanation
+
+This type of manifest describes test vectors to create.
+Processing these test scenarios will result in [encryption materials](../structures.md#encryption-materials).
+
+Each test scenario includes all necessary instructions to construct
+an [encryption materials request](../cmm-interface.md#encryption-materials-request).
+This includes all necessary inputs including algorithm suite, encryption context, and keyrings/CMMs.
+
+## Reference-level Explanation
+
+### manifest
+
+Map identifying the manifest.
+
+- `type` : Identifies the manifest as an AWS Encryption SDK message encryption manifest.
+  - Must be `aws-mpl-encrypt`
+- `version` : Identifies the version of this feature document that describes the manifest.
+
+#### keys
+
+URI identifying a [keys manifest](./keys-manifest.md) to use with all tests.
+
+### tests
+
+Map object mapping a test case ID to a test case description
+that describes how to generate an [encryption materials request](../cmm-interface.md#encryption-materials-request).
+Optional members on [encryption materials request](../cmm-interface.md#encryption-materials-request)
+are optional in the test.
+
+Map object mapping a test case ID to a test case description
+that describes how to generate an [encryption materials request](../cmm-interface.md#encryption-materials-request).
+Optional members on [encryption materials request](../cmm-interface.md#encryption-materials-request)
+are optional in the test.
+
+- type : The type of test
+  - Allowed Values
+    - positive-keyring : A successful request for encrypt materials that should also result in a successful decrypt
+    - negative-decrypt-keyring: A successful request for encrypt materials that should result in a failure to decrypt
```

Comment:
i.e. some implementation successfully creates a ciphertext with this test case, but we expect decrypting with this test case to fail? What's a useful example here?

Reply:
It is not that one needs to succeed to fail. This test means that we ned to have encrypted something in order to then test the failure case.